<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="hu_HU" sourcelanguage="en_US">
<context>
    <name>ExportDialog</name>
    <message>
        <location filename="../export_plugin_dialog.py" line="51"/>
        <source>Missing file</source>
        <translation>Hiányzó fájl</translation>
    </message>
    <message>
        <location filename="../export_plugin_dialog.py" line="51"/>
        <source>GeoPackage to be exported not found</source>
        <translation>Exportálandó GeoPackage nem található</translation>
    </message>
</context>
<context>
    <name>GmlExporter</name>
    <message>
        <location filename="../gml_exporter.py" line="90"/>
        <source>Unsupported geometry type: </source>
        <translation>Nem kezelt geometria típus</translation>
    </message>
    <message>
        <location filename="../gml_exporter.py" line="167"/>
        <source>CRITICAL error</source>
        <translation>KRITIKUS hiba</translation>
    </message>
    <message>
        <location filename="../gml_exporter.py" line="167"/>
        <source>Cannot open GeoPackage: </source>
        <translation>Nem lehet megnyitni a GeoPackage-t</translation>
    </message>
    <message>
        <location filename="../gml_exporter.py" line="203"/>
        <source>INFO</source>
        <translation>INFO</translation>
    </message>
    <message>
        <location filename="../gml_exporter.py" line="203"/>
        <source>Succesful GML export</source>
        <translation>Sikeres GML export</translation>
    </message>
</context>
<context>
    <name>GmlImportExport</name>
    <message>
        <location filename="../import_export_plugin.py" line="59"/>
        <source>&amp;GML import / export</source>
        <translation>&amp;GML import / export</translation>
    </message>
    <message>
        <location filename="../import_export_plugin.py" line="146"/>
        <source>Import</source>
        <translation>Import</translation>
    </message>
    <message>
        <location filename="../import_export_plugin.py" line="148"/>
        <source>Export</source>
        <translation>Export</translation>
    </message>
    <message>
        <location filename="../import_export_plugin.py" line="150"/>
        <source>Load</source>
        <translation>Betöltés</translation>
    </message>
    <message>
        <location filename="../import_export_plugin.py" line="152"/>
        <source>Validate</source>
        <translation>Ellenőrzés</translation>
    </message>
    <message>
        <location filename="../import_export_plugin.py" line="178"/>
        <source>Warning</source>
        <translation>Figyelmeztetés</translation>
    </message>
    <message>
        <location filename="../import_export_plugin.py" line="178"/>
        <source>Project is not empty! Do you want to add layers from GML?</source>
        <translation>A projekt nem üres. Hozzáadja a rétegeket a GML fájlból?</translation>
    </message>
</context>
<context>
    <name>GmlImporter</name>
    <message>
        <location filename="../gml_importer.py" line="42"/>
        <source>GML identifier: </source>
        <translation>GML azonosító:</translation>
    </message>
    <message>
        <location filename="../gml_importer.py" line="46"/>
        <source>XSD version: </source>
        <translation>XSD verzió:</translation>
    </message>
    <message>
        <location filename="../gml_importer.py" line="96"/>
        <source> layer </source>
        <translation>réteg</translation>
    </message>
    <message>
        <location filename="../gml_importer.py" line="104"/>
        <source>Succesful GML import</source>
        <translation>Sikeres GML import</translation>
    </message>
    <message>
        <location filename="../gml_importer.py" line="104"/>
        <source> imported</source>
        <translation>importálva</translation>
    </message>
    <message>
        <location filename="../gml_importer.py" line="114"/>
        <source>ERROR</source>
        <translation>HIBA</translation>
    </message>
    <message>
        <location filename="../gml_importer.py" line="114"/>
        <source> open/read error </source>
        <translation>Nyitási/olvasási hiba</translation>
    </message>
</context>
<context>
    <name>GmlValidator</name>
    <message>
        <location filename="../gml_validator.py" line="52"/>
        <source>CRITICAL error</source>
        <translation>KRITIKUS hiba</translation>
    </message>
    <message>
        <location filename="../gml_validator.py" line="26"/>
        <source>Python lxml package not found, please install it.</source>
        <translation>Az lxml Python csomag nem található, kérem telepítse.</translation>
    </message>
    <message>
        <location filename="../gml_validator.py" line="39"/>
        <source>Schema not found: </source>
        <translation>Séma nem található:</translation>
    </message>
    <message>
        <location filename="../gml_validator.py" line="45"/>
        <source>Error in schema: </source>
        <translation>Hiba a sémában:</translation>
    </message>
    <message>
        <location filename="../gml_validator.py" line="52"/>
        <source>Invalid or missing GML file: </source>
        <translation>Hibás vagy hiányzó GML fájl:</translation>
    </message>
    <message>
        <location filename="../gml_validator.py" line="58"/>
        <source>ERROR</source>
        <translation>HIBA</translation>
    </message>
    <message>
        <location filename="../gml_validator.py" line="58"/>
        <source>Error in GML: </source>
        <translation>Hiba a GML-ben:</translation>
    </message>
    <message>
        <location filename="../gml_validator.py" line="61"/>
        <source>INFO</source>
        <translation>INFO</translation>
    </message>
    <message>
        <location filename="../gml_validator.py" line="61"/>
        <source> is valid to : </source>
        <translation> megfelel ennek:</translation>
    </message>
</context>
<context>
    <name>GpkgLoader</name>
    <message>
        <location filename="../gpkg_loader.py" line="54"/>
        <source>Cannot load layer: </source>
        <translation>Nem lehet betölteni a réteget:</translation>
    </message>
</context>
<context>
    <name>ImportDialog</name>
    <message>
        <location filename="../import_plugin_dialog.py" line="34"/>
        <source>Empty field</source>
        <translation>Üres mező</translation>
    </message>
    <message>
        <location filename="../import_plugin_dialog.py" line="34"/>
        <source>Fill both fields</source>
        <translation>Töltse ki mindkét mezőt</translation>
    </message>
    <message>
        <location filename="../import_plugin_dialog.py" line="38"/>
        <source>Misisng file</source>
        <translation>Hiányzó fájl</translation>
    </message>
    <message>
        <location filename="../import_plugin_dialog.py" line="38"/>
        <source>GML file does not exist</source>
        <translation>GML fájl nem létezik</translation>
    </message>
    <message>
        <location filename="../import_plugin_dialog.py" line="42"/>
        <source>Existing file</source>
        <translation>Létező fájl</translation>
    </message>
    <message>
        <location filename="../import_plugin_dialog.py" line="42"/>
        <source>GeoPackage already exists, overwrite?</source>
        <translation>Geopackage fájl már létezik, felülírja?</translation>
    </message>
</context>
<context>
    <name>LoadDialog</name>
    <message>
        <location filename="../load_plugin_dialog.py" line="31"/>
        <source>Empty field</source>
        <translation>Üres mező</translation>
    </message>
    <message>
        <location filename="../load_plugin_dialog.py" line="31"/>
        <source>GeoPackage field is obligatory</source>
        <translation>GeoPackage mező kitöltése kötelező</translation>
    </message>
</context>
<context>
    <name>ValidateDialog</name>
    <message>
        <location filename="../validate_plugin_dialog.py" line="31"/>
        <source>Empty field</source>
        <translation>Üres mező</translation>
    </message>
</context>
</TS>
